framework/src/Volo.Abp.Caching/Volo/Abp/Caching/DistributedCache.cs

```diff
@@ -133,14 +142,25 @@ protected virtual void SetDefaultOptions()
         /// Gets a cache item with the given key. If no cache item is found for the given key then returns null.
         /// </summary>
         /// <param name="key">The key of cached item to be retrieved from the cache.</param>
+        /// <param name="considerUow">This will store the cache in the current unit of work until the end of the current unit of work does not really affect the cache.</param>
         /// <param name="hideErrors">Indicates to throw or hide the exceptions for the distributed cache.</param>
         /// <returns>The cache item, or null.</returns>
         public virtual TCacheItem Get(
             TCacheKey key,
+            bool considerUow = false,
```

Comment:
It would be better to add an optional parameter to the end of the parameter list. In this case, types (bool) is same and if someone set the `hideErrors` before, they will be set the `considerUow` when they upgrade the ABP Framework, which will be an implicit breaking change.

Same for other methods.